dts/arm/renesas/smartbond/da1469x.dtsi

```diff
@@ -86,6 +86,52 @@
 			clock-src = <&rc32k>;
 			status = "okay";
 		};
+
+		regulators {
+			compatible = "renesas,smartbond-regulator";
+			vdd: VDD {
+				regulator-init-microvolt = <900000>;
+				regulator-boot-on;
+				renesas,regulator-sleep-ldo;
+				renesas,regulator-dcdc-vbat-high;
+				renesas,regulator-dcdc-vbat-low;
+			};
+			vdd_clamp: VDD_CLAMP {
+				regulator-boot-on;
+				regulator-always-on;
+				regulator-init-microvolt = <706000>;
+			};
+			vdd_sleep: VDD_SLEEP {
+				regulator-boot-on;
+				regulator-init-microvolt = <750000>;
```

Comment:
are all these init values board specific? why are they defined in soc dts files?

Reply:
They are not board specific, they can be changed for board but they are soc specific values used here to achieve lowest power consumption.